tests/aws/services/stepfunctions/v2/test_stepfunctions_v2.py

```diff
@@ -831,3 +831,4 @@ def check_invocations():
 
     # clean up
     cleanup(sm_arn, state_machines_before, aws_client.stepfunctions)
+    # TODO also clean up other resources (like secrets)
```

Comment:
Since this is something that's definitely broken, I'd use `FIXME` here instead of `TODO`. But I don't know if we have a proper decision on when to use which here.

Reply:
Yeah, that's a subtle difference where we don't have a clear structure yet. I do agree that a FIXME would maybe fit better, but for the sake of unblocking master asap, I would keep it like this for now.